rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/DeliverableAnalyzerReportEndpoint.java

```diff
@@ -74,6 +78,57 @@ public interface DeliverableAnalyzerReportEndpoint {
 
     String GET_DEL_AN_REPORT_LABEL_HISTORY = "Gets the label history of this deliverable analyzer report";
 
+    String GET_ALL_DESC = "Gets all deliverable analyzer reports.";
+
+    /**
+     * {@value GET_ALL_DESC}
+     *
+     * @param pageParams
+     * @return
+     */
+    @Operation(
+            summary = GET_ALL_DESC,
+            responses = { @ApiResponse(
+                    responseCode = SUCCESS_CODE,
+                    description = SUCCESS_DESCRIPTION,
+                    content = @Content(
+                            schema = @Schema(implementation = SwaggerPages.DeliverableAnalyzerReportPage.class))),
+                    @ApiResponse(
+                            responseCode = INVALID_CODE,
+                            description = INVALID_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))),
+                    @ApiResponse(
+                            responseCode = SERVER_ERROR_CODE,
+                            description = SERVER_ERROR_DESCRIPTION,
+                            content = @Content(schema = @Schema(implementation = ErrorResponse.class))) })
+    @GET
+    @TimedMetric
+    Page<DeliverableAnalyzerReport> getAll(@Valid @BeanParam PageParameters pageParams);
+
+    String GET_SPECIFIC_DESC = "Gets specific deliverable analyzer report.";
```

Comment:
Move please to top of file, where are all other endpoints' descriptions

Reply:
I'll do the exact oposite.

Reply:
DeliverableAnalyzerReportEndpointTest, line 174, remove last remaining TODO please